.travis.yml

```diff
@@ -10,6 +10,6 @@ rvm:
 
 script:
 - cd service_management/azure && bundle install && bundle exec rake test:unit
+- if [ "$TRAVIS_ASM_INTEG_RECORDED" == "true" ] ; then bundle exec rake test:recorded:affinity_group ; fi
```

Comment:
Travis CI build definition sets this environment variable. Line 132 [Job 504.1](https://travis-ci.org/Azure/azure-sdk-for-ruby/jobs/119311056)